app/services/publisher/adapters/cloud_platform.rb

```diff
@@ -59,14 +59,6 @@ def completed; end
 
       private
 
-      def first_published?
```

Comment:
Removed this method as it was not being used anywhere, but it gave me the clue how to do it to see if the email must trigger or not 😅